packages/dds/tree/src/core/rebase/utils.ts

```diff
@@ -276,6 +289,8 @@ export function rebaseBranch<TChange>(
 			targetCommits,
 			sourceCommits,
 		},
+		sourceBranchLength: sourcePath.length,
+		rebaseDistance: targetCommits.length - (sourcePath.length - sourceSet.size),
```

Comment:
@yann-achard-MS 

I would like to double check if this corresponds to your definition of `rebaseDefinition` (number of commits that we are rebasing over).

Reply:
I presume you mean "definition of `rebaseDistance`." As I mentioned on Friday, I was thinking we would just return `targetCommitIndex+1`, but similarly to above, this also makes sense. I will get back to you on that.

Reply:
See comment above.